docs/COMMUNITIES.md

```diff
@@ -15,8 +15,15 @@ Many language communities are self organising and have their contact channels on
    * [Web](https://dhivehi.ai/)
    * [Discord](https://discord.com/invite/5ZdTHsN8Mn)
    * [Matrix](https://app.element.io/#/room/!WNiJFJTBpvYqMrDWir:matrix.org)
+* Esperanto (`eo`):
+   * [Matrix](https://chat.mozilla.org/#/room/#Esperanto:mozilla.org)
+   * [GitHub](https://github.com/parolrekonado)
+   * [Website](https://parolrekonado.github.io/)
```

Comment:
Is this the correct link?

Reply:
Looks correct to me, what are you concerned about?

Reply:
When I opened it 2 days ago it showed the theme documentation, but there indeed was work done recently. All good now!